frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -195,10 +195,9 @@ function handleProjectLocationClick() {
 
 async function onAssistanceRequest(values: GenericObject) {
   try {
-    const formattedData = Object.assign(
+    const submissionData = Object.assign(
```

Comment:
Memory management: Is there a particular reason why we're using Object.assign to construct this object? Looking at the arguments, it looks lke we're fully defining the objects ourselves anyways in the two merge arguments, which means this could be simplified down to directly instantiating the object we want to have.

Reply:
Holdover from old code being a little too worried about modifying the object values by reference. After multiple changes this is no longer an issue. Directly instantiating object now.